test/macros.jl

```diff
@@ -452,3 +452,32 @@ facts("[macros] Issue #621") do
     con = @QuadConstraint(a+q*3 <= 0)
     @fact conToStr(con) --> "3 x² + x + 1 $leq 0"
 end
+
+facts("[macros] @defVars and @addConstraints") do
+    m = Model()
+    @defVars m begin
+        0 ≤ x ≤ 1
```

Comment:
Maybe have a variable with indicies and nontrivial bounds, just to be safe with parsing?